redisinsight/ui/src/components/monitor/MonitorWrapper.tsx

```diff
@@ -38,18 +41,28 @@ const MonitorWrapper = () => {
 
   return (
     <section className={styles.monitorWrapper} data-testid="monitor-container">
-      <MonitorHeader handleRunMonitor={handleRunMonitor} />
-      <Monitor
-        items={items}
-        error={error}
-        isStarted={isStarted}
-        isPaused={isPaused}
-        isRunning={isRunning}
-        isShowCli={isShowCli}
-        isSaveToFile={isSaveToFile}
-        isShowHelper={isShowHelper}
-        handleRunMonitor={onRunMonitor}
-      />
+      <FeatureFlagComponent
+        name={FeatureFlags.disabledByEnv}
+        enabledByDefault
+        otherwise={(
+          <div style={{ display: 'grid', placeContent: 'center', height: '100%' }}>
+            <EuiTextColor color="accent">Profiler not implemented in this environment.</EuiTextColor>
```

Comment:
Sorry to be pedantic, but can we change the message to "Monitor not supported in this environment." ? 

Mentioning profiler (which is a Redis Insight desktop feature) when trying to run the Redis command `Monitor` seems misleading. Saying something isn't implemented makes it sound like we haven't gotten around to writing the code, whereas saying it isn't supported leaves it more open ended.